src/bundler.ts

```diff
@@ -191,14 +218,20 @@ class Bundler {
 
     const imprt = importMap.get(resolvedUrl);
     if (imprt) {
```

Comment:
for future reference, no action needed: where the logical name is reserved or already taken I prefer adding something like `_`, or making a long name like `analzedImport` than mangling the name.